EXOSIMS/Completeness/BrownCompleteness.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, Nplanets=1e8, **specs):
         self.Nplanets = int(Nplanets)
        
         # get path to completeness interpolant stored in a pickled .comp file
-        self.filename = self.PlanetPopulation.__class__.__name__ + self.PlanetPhysicalModel.__class__.__name__ + self.__class__.__name__
+        self.filename = self.PlanetPopulation.__class__.__name__ + self.PlanetPhysicalModel.__class__.__name__ + self.__class__.__name__ + str(self.Nplanets) + self.PlanetPhysicalModel.whichPlanetPhaseFunction

```

Comment:
This is an ICD-level change.  Please update the ICD accordingly.  Also, do we have a separate ICD section on cache file naming conventions?  If not, we should create one.